priv/templates/nimble_template/README.md.eex

```diff
@@ -8,72 +8,76 @@
 
 ### Erlang & Elixir
 
-* Erlang <%= erlang_version %>
+- Erlang <%= erlang_version %>
 
-* Elixir <%= elixir_version %>
+- Elixir <%= elixir_version %>
 
-* Recommended version manager.
+- Recommended version manager.
 
   - [asdf](https://github.com/asdf-vm/asdf)
   - [asdf-erlang](https://github.com/asdf-vm/asdf-erlang)
   - [asdf-elixir](https://github.com/asdf-vm/asdf-elixir)
 
 ### Development
 
-* Install [Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
+- Install [Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
 
-* Setup and boot the Docker containers:
+- Setup and boot the Docker containers:
 
   ```sh
   make docker_setup
   ```
 
-* Install Elixir dependencies:
+- Install Elixir dependencies:
 
   ```sh
   mix deps.get
   ```
 <%= if web_project? do %>
-* Install Node dependencies:
+- Install Node dependencies:
 
   ```sh
   npm install --prefix assets
   ```
 <% end %>
-* Setup the databases:
+- Setup the databases:
```

Comment:
This issue has been fixed by Junan as I remember https://github.com/nimblehq/elixir-templates/pull/111, could you rebase this PR @longnd 🙏

Reply:
yes, rebased.